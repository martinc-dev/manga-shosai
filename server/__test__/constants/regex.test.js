const regexConstants = require('app/constants/regex')

describe('constants/regex', () => {
  it('Matches snapshots', () => {
    const actual = regexConstants

    expect(actual).toMatchSnapshot()
  })
})
