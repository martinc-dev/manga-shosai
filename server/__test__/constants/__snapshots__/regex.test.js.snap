// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants/regex Matches snapshots 1`] = `
Object {
  "hiddenFilenameRegex": "^\\\\.",
  "thumbnailTokenRegex": "[A-Za-z0-9]{16}$",
  "uuidV4Regex": "[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-4[0-9A-Fa-f]{3}-[89ABab][0-9A-Fa-f]{3}-[0-9A-Fa-f]{12}$",
}
`;
